src/listener.ts

```diff
@@ -143,6 +143,13 @@ export const getRequestListener = (
     // so generate a pseudo Request object with only the minimum required information.
     const req = newRequest(incoming)
 
+    // Detect if request was aborted.
+    incoming.socket.on('close', () => {
+      if (incoming.aborted) {
```

Comment:
From the nodejs docs it appears that the `aborted` property is deprecated, and as a replacement, we should check the `destroyed` (https://nodejs.org/dist/latest-v20.x/docs/api/http.html#messageaborted).

For some reason if i use it, it makes most of the tests fail, do not know if this is something related with supertest...

Reply:
Resolved, had to listen to the close event on the response and check the `incoming.socket.destroyed`